templates/home.html

```diff
@@ -0,0 +1,37 @@
+{% extends "base.html" %}
+
+{% block title %}Home{% endblock %}
+
+{% block content %}
+<ul class="nav">
+    {% if user.is_authenticated %}
+     <li class="nav-item">
+            <a class="nav-link active" href="{% url 'logout' %}">Logout</a>
+        </li>
+
+    {% else %}
+    {% if request.path != '/auth/login/' %}
```

Comment:
if statement should be
```
{% if not user.is_authenticated %}
```

Reply:
I've tried but this condition didn't show the desired result.